ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```diff
@@ -1,7 +1,7 @@
 base_images:
   assisted_service:
-    name: "4.7"
-    namespace: ocp
+    name: assisted
+    namespace: ocp-kni
```

Comment:
`assisted_service` tag does not seem to exist yet in the imagestream - it's what the faiing rehearsals signal. This will fix itself automatcially once the image gets built and promoted from assisted-service, but until that happen, CI jobs fro the `assisted-test-infra` repo will be broken. Just FYI.